sync/src/synchronizer/block_fetcher.rs

```diff
@@ -113,17 +112,12 @@ impl BlockFetcher {
         debug_assert!(best_known_header.number() > fixed_last_common_header.number());
 
         let mut index_height = fixed_last_common_header.number();
-        // Read up to 128, get_ancestor may be as expensive
-        // as iterating over ~100 entries anyway.
-        let mut fetch = Vec::with_capacity(PER_FETCH_BLOCK_LIMIT);
+        let mut fetch = Vec::with_capacity(MAX_BLOCKS_IN_TRANSIT_PER_PEER);
```

Comment:
Is `PER_FETCH_BLOCK_LIMIT` no long used? It's better to remove the const as well.

Reply:
yes, removed already